app/services/adapters/structs/applicant.rb

```diff
@@ -1,14 +1,7 @@
 module Adapters
   module Structs
     class Applicant < BaseStructAdapter
-      # NOTE: for MVP this is always true, as we are doing
-      # screening of applicants with DWP. This will change
-      # post-MVP but we don't know yet how exactly.
-      def passporting_benefit
-        true
-      end
-
-      # Same as above, for MVP will always be `yes`
+      # For MVP this will always be `yes`
       # This is not part of the application JSON for now.
       def has_nino
         YesNoAnswer::YES
```

Comment:
Is this OK to leave hardcoded as Yes?

Reply:
It is not used really, but post-MVP we might use it. I can remove it to avoid confusion.
I think it is one of those attributes that can be inferred from other bits of information (like `applicant.nino.presence?`)

Reply:
👍 Ok, I didn't mind either way! Just checking.

Reply:
I've removed it 😄